core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -87,6 +101,18 @@ var PreferencesPage = function() {
     expect(selectedLanguageElement.getText()).toEqual(language);
   };
 
+  this.expectPreferredAudioLanguageToBe = function(language) {
+    var selectedLanguageElement = preferredAudioLanguageSelector.element(
```

Comment:
This should also go at the top of the file (near lines 23-33)

Reply:
Done!